frontend/src/components/widgets/CameraImageInput/WebcamComponent.tsx

```diff
@@ -54,10 +54,15 @@ const WebcamComponent = ({ handleCapture, width }: Props): ReactElement => {
   const videoRef = useRef<Webcam>(null)
   const [facingMode, setFacingMode] = useState(FacingMode.USER)
 
+  const [loading, setLoading] = useState(true)
+  const display = loading ? "none" : "block"
+
   function capture(): void {
-    if (videoRef.current !== null) {
+    if (videoRef.current !== null && !loading) {
       const imageSrc = videoRef.current.getScreenshot()
       handleCapture(imageSrc)
+    } else {
+      alert("Please wait... The webcam is loading.")
```

Comment:
I was never able to hit this part of the code so not sure if we should include this but i just wrote this to be safe.